drake-torch-noetic.dockerfile

```diff
@@ -46,11 +48,16 @@ RUN apt-get update && apt-get install -qy \
     ros-noetic-roslint \
     # ros-noetic-gazebo-ros \
     ros-noetic-async-web-server-cpp \
-    ros-noetic-realsense2-camera \
+    ros-noetic-realsense2-camera
```

Comment:
Can we pin the version here?

Reply:
NEVER! [😊 i'll know if and when it breaks anything

Reply:
We will have a bigger conversation around this some time, for now, this is ok :+1: